_docs/what-is-thecloudnatives.md

```diff
@@ -1,32 +1,21 @@
 ---
-title: "What is Crisp?"
+title: "What is FikaWorks?"
 prev_section: index.html
 next_section: how-thecloudnatives-works.html
```

Comment:
Should we rename this document to match new name? `how-thecloudnatives-works.html > how-fikaworks-works.html`

```suggestion
next_section: how-fikaworks-works.html
```